client/src/components/NewLocationReview.jsx

```diff
@@ -0,0 +1,275 @@
+import { inputRef, useEffect } from 'react';
```

Comment:
You imported useState twice, you only need to import once... 
you also have multiple imports from react, you can just do 
`import React, { useState, useRef, useEffect } from 'react';`

Reply:
There's only one import of useState.